Mailbox/Mailbox.cs

```diff
@@ -1,4 +1,26 @@
-﻿namespace Mailbox
+﻿using System;
+
+namespace Mailbox
 {
+    public class Mailbox 
+    {
+        public ValueTuple<int, int> Location { get; }
```

Comment:
ValueTuple syntax can be simplified
```suggestion
        public (int X, int Y) Location { get; }
```